exec/cnex/format.h

```diff
@@ -0,0 +1,23 @@
+#ifndef FORMAT_H
+#define FORMAT_H
+#include <stdint.h>
+
+#include "exec.h"
```

Comment:
You don't need to include all of `exec.h` here. A forward class declaration would do just fine.

Reply:
Removed include.